maintainers/maintainer-list.nix

```diff
@@ -4199,6 +4199,12 @@
     githubId = 74379;
     name = "Florian Pester";
   };
+  fmoda3 = {
+    email = "fmoda3@mac.com";
+    github = "fmoda3";
+    githubId = 1746471;
+    name = "Frank Moda III";
+  };
```

Comment:
Please make this into a separate commit:

```
maintainers: add <name>
```

Reply:
Pulled this out into a separate commit